### DISCUSSION TITLE
Fullstack .net

### DISCUSSION THREAD

**Discussion by DennerLevi** [balta-io/forum#665] [Tira Dúvidas] [answered]
Comecei o curso recentemente e comecei pela apresentação do curso e depois estruturando o projeto. Logo depois eu vi que foram adicionados novos vídeos e fiquei na duvida : esses cursos são continuções? ou eu paro tudo e começo da área de pedidas - back end(NOVO)?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @DennerLevi, como vai?

Sim, adicionamos dois novos módulos no curso Fullstack, e nas próximas semanas vamos adicionar mais um. Não é um novo curso (Embora sejam módulos bem completos), todos eles fazem parte do curso Fullstack.

Os módulos estão na ordem que precisam ser seguidos dentro do curso... não há necessidade de parar tudo ou mesmo de começar pela área de pedidos... Começa do primeiro módulo e vai até o fim!
